libs/types/src/cdf/README.md

```diff
@@ -0,0 +1,18 @@
+# Common Date Format (CDF) Types
+
+The National Institute for Standards and Technology (NIST) has defined a set of
+[Common Data Format (CDF) specifications](https://www.nist.gov/itl/voting/interoperability)
+for various types of election data.
+
+In order to support inputting/outputting data in these formats, we auto-generate
+types and Zod schemas for each of the CDF specs using our
+[CDF Schema Builder](../../../cdf-schema-builder/).
+
+However, in certain cases we have needed to add extensions and/or constraints to
+the CDF specs based on the needs of our system. In each subdirectory, you will
+find a `vx-schema.json` file that contains the VotingWorks version of the CDF
+schema. The original NIST schemas can be found in the
+[data directory](../../data/). Note that due to some weirdness with how NIST
+provides their schemas, we use both the XSD (XML) format of the schema, which
+includes documentation, and the JSON format, which includes the information we
+need to generate the types.
```

Comment:
This isn't quite true yet, but I'm assuming that if we like this approach, @adghayes or I will update the other CDF subdirectories accordingly